modules/howtos/pages/full-text-searching-with-sdk.adoc

```diff
@@ -195,6 +196,17 @@ You can tell the Search Engine to include the full content of a certain number o
 include::example$Search.java[tag=fields,indent=0]
 ----
 
+=== Collections
```

Comment:
Noticed we hadn't covered this, I will create a ticket to update the other SDKs, but doing it now for Java since I am already here:

This was the change:
https://github.com/couchbase/couchbase-jvm-clients/commit/a9c325bc1cf7a3f0fb7fe08a31a4b04f1c2a8c47